YangJsonTools/swagger.py

```diff
@@ -520,9 +525,11 @@ def gen_api_node(node, path, apis, definitions, config = True):
             else:
                 schema = None
 
-        apis['/config'+str(path)] = print_api(node, config, schema, path)
+        apis['/config'+str(path).replace(" ", "_")] = print_api(node, config, schema, path.replace(" ", "_"))
```

Comment:
What does this change do? Could you add a brief comment explaining?

Reply:
this removes the spaces in this outputted path:
```json
{
        "/config/context/topology/{uuid}/node/{node_uuid}/owned-node-edge-point/{owned_node_edge_point_uuid}/aggregated-node-edge-point/{topology_id node_id owned_node_edge_point_id}/": {
            "get": {
                "responses": {
                    "200": {
                        "description": "Successful operation",
                        "schema": {
                            "$ref": "#/definitions/owned-node-edge-point-ref"
                        }
                    },
                    "400": {
                        "description": "Internal Error"
                    }
                },
                "description": "Retrieve operation of resource: aggregated-node-edge-point",
                "parameters": [
                    {
                        "description": "ID of uuid",
                        "required": true,
                        "type": "string",
                        "name": "uuid",
                        "in": "path"
                    },
                    {
                        "description": "ID of node_uuid",
                        "required": true,
                        "type": "string",
                        "name": "node_uuid",
                        "in": "path"
                    },
                    {
                        "description": "ID of owned_node_edge_point_uuid",
                        "required": true,
                        "type": "string",
                        "name": "owned_node_edge_point_uuid",
                        "in": "path"
                    },
                    {
                        "description": "ID of topology_id node_id owned_node_edge_point_id",
                        "required": true,
                        "type": "string",
                        "name": "topology_id node_id owned_node_edge_point_id",
                        "in": "path"
                    }
                ],
                "produces": [
                    "application/json"
                ],
                "tags": [
                    "topology"
                ],
                "summary": "Retrieve aggregated-node-edge-point by ID",
                "consumes": [
                    "application/json"
                ],
                "operationId": "retrieve_context_topology_node_owned-node-edge-point_aggregated-node-edge-point_aggregated-node-edge-point_by_id"
            }
        }
}
```